README.md

```diff
@@ -27,23 +29,211 @@ You can use the package as follows:
 ```python
 from santoku.slack import SlackBotHandler
 
-slack_bot = SlackBotHandler.from_aws_secrets_manager(...)
+slack_bot = SlackBotHandler.from_aws_secrets_manager("your_secret")
+```
+
+## Content
+
+The package `santoku` contains several subpackages: `aws`, `google`, `salesforce` and `slack`. Each subpackage provides connection to different external services and are formed by a collection of modules, where each module consists of handlers for more specific services. Each handler class has unit tests to ensure the correct behaviour of the methods of these classes.
+
+### AWS
+
+The connection to AWS has been done through [boto3](https://github.com/boto/boto3). We provide wrappers of the `boto3` sdk to make easy the operations to interact with different services.
+
+The use of this subpackage requires having AWS credentials somewhere. We provide flexibility to either keep credentials in AWS credentials/configuration file or to pass them directly as arguments in the initializer of each handler class. More info on AWS configurations and credentials [here](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html)
+
+The unit tests in this subpackage implement mocks to the AWS services and do not pretended to access or modify the environment of your real account.

```

Comment:
I would extend this a bit:
"In order to have safer unit tests for the AWS subpackage, we use [moto](link), a mocking library for most AWS services, which allows our methods to interact with a fully mocked version of the AWS environment via decorators while not needing an actual connection to the internet or a test AWS account."